x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/fulltext/Kql.java

```diff
@@ -35,7 +37,9 @@ public class Kql extends FullTextFunction {
 
     @FunctionInfo(
         returnType = "boolean",
-        preview = true,
+        appliesTo = {
+            @FunctionAppliesTo(lifeCycle = FunctionAppliesToLifecycle.PREVIEW, version = "9.0.0"),
+            @FunctionAppliesTo(lifeCycle = FunctionAppliesToLifecycle.GA, version = "9.1.0") },
```

Comment:
Why does this come with 2 markers? Isn't the `GA in 9.1` information sufficient?

Also, what about `8.19`? I think most that's going live in 9.1 is also going live in 8.19, no?